ui/desktop/app/styles/app.scss

```diff
@@ -96,7 +96,7 @@
     margin-bottom: sizing.rems(m);
 
     legend {
-      @extend .rose-form-label;
+      @extend .hds-form-label !optional;
```

Comment:
trying to understand the context- could you explain why this line is needed?

Reply:
Thanks for flagging, this can actually be removed. Originally, the terminal returned an err and suggested adding `!optional` but realizing it's not necessary as the styling belongs to a rose component that will likely be retired in the future.